pkg/gofr/datasource/dgraph/logger.go

```diff
@@ -35,7 +35,7 @@ func (ql *QueryLog) PrettyPrint(logger Logger) {
 	logger.Debug(formattedLog)
 }
 
-// clean replaces multiple consecutive whitespace characters with a single space and trims leading/trailing whitespace
+// clean replaces multiple consecutive whitespace characters with a single space and trims leading/trailing whitespace.
 func clean(query string) string {
 	query = regexp.MustCompile(`\s+`).ReplaceAllString(query, " ")
```

Comment:
Can we move the regex as a variable, since logs are printed frequently it may increase the time to execute a query.

Reply:
sure moved that regex to a variable